.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+## Steps to reproduce
+
+How'd you do it?
+
+1. ...
+2. ...
+
+This section should also tell us any relevant information about the
+environment; for example, if an exploit that used to work is failing,
+tell us the victim operating system and service versions.
```

Comment:
I feel like a little more detail here wouldn't hurt.  For example, if they have the name of the module used, that is critical.  If they have the exact steps and output from `msfconsole`, that would be helpful.  If they have logs, great.